app/src/main/java/com/example/vmedvediev/kotlintimer/util/PrefUtil.kt

```diff
@@ -4,70 +4,57 @@ import android.content.Context
 import android.preference.PreferenceManager
 import com.example.vmedvediev.kotlintimer.TimerActivity
 
-class PrefUtil {
-    companion object {
 
-        private const val TIMER_LENGTH_ID = "com.example.timer.timer_length"
-        fun getTimerLength(context: Context): Int{
-            val preferences = PreferenceManager.getDefaultSharedPreferences(context)
-            return preferences.getInt(TIMER_LENGTH_ID, 1)
-        }
+object PrefUtil {
 
+        private const val TIMER_LENGTH_ID = "com.example.timer.timer_length"
         private const val PREVIOUS_TIMER_LENGTH_SECONDS_ID = "com.example.timer.previous_timer_length_seconds"
+        private const val TIMER_STATE_ID = "com.example.timer.timer_state"
+        private const val SECONDS_REMAINING_ID = "com.example.timer.seconds_remaining"
+        private const val ALARM_SET_TIME_ID = "com.example.timer.backgrounded_time"
 
-        fun getPreviousTimerLengthSeconds(context: Context): Long{
-            val preferences = PreferenceManager.getDefaultSharedPreferences(context)
-            return preferences.getLong(PREVIOUS_TIMER_LENGTH_SECONDS_ID, 0)
-        }
+        private fun getPreferences(context: Context) = PreferenceManager.getDefaultSharedPreferences(context)
 
-        fun setPreviousTimerLengthSeconds(seconds: Long, context: Context){
-            val editor = PreferenceManager.getDefaultSharedPreferences(context).edit()
-            editor.putLong(PREVIOUS_TIMER_LENGTH_SECONDS_ID, seconds)
-            editor.apply()
-        }
+        private fun getPreferencesEditor(context: Context) = PreferenceManager.getDefaultSharedPreferences(context).edit()
 
+        fun getTimerLength(context: Context): Int = getPreferences(context).getInt(TIMER_LENGTH_ID, 1)
 
-        private const val TIMER_STATE_ID = "com.example.timer.timer_state"
+        fun getPreviousTimerLengthSeconds(context: Context): Long = getPreferences(context).getLong(PREVIOUS_TIMER_LENGTH_SECONDS_ID, 0)
+
+        fun getAlarmSetTime(context: Context): Long = getPreferences(context).getLong(ALARM_SET_TIME_ID, 0)
+
+        fun getSecondsRemaining(context: Context): Long = getPreferences(context).getLong(SECONDS_REMAINING_ID, 0)
 
         fun getTimerState(context: Context): TimerActivity.TimerState{
-            val preferences = PreferenceManager.getDefaultSharedPreferences(context)
-            val ordinal = preferences.getInt(TIMER_STATE_ID, 0)
+            val ordinal = getPreferences(context).getInt(TIMER_STATE_ID, 0)
             return TimerActivity.TimerState.values()[ordinal]
         }
 
-        fun setTimerState(state: TimerActivity.TimerState, context: Context){
-            val editor = PreferenceManager.getDefaultSharedPreferences(context).edit()
-            val ordinal = state.ordinal
-            editor.putInt(TIMER_STATE_ID, ordinal)
-            editor.apply()
+        fun setPreviousTimerLengthSeconds(seconds: Long, context: Context){
```

Comment:
Change the order of function arguments:
the first -  should be abstract objects or interfaces, 
the others - common properties